src/TraceEvent/DynamicTraceEventParser.cs

```diff
@@ -48,6 +49,8 @@ public DynamicTraceEventParser(TraceEventSource source)
             registeredParser = new RegisteredTraceEventParser(source, true);
             // But cause any of its new definitions to work on my subscriptions.  
             registeredParser.NewEventDefinition = OnNewEventDefintion;
+            // make an eventPipeTraceEventParser to resolve EventPipe events
+            eventPipeTraceEventParser = new EventPipeTraceEventParser(source, dontRegister: true);
```

Comment:
I do NOT wish to have DynamicTraceEventParser (which is basically knows about inline manifest, tracelogging and classic ETW based events (but they are all ETW), to have any relationship with EventPipeTraceEventParser (which is its own format and technically has nothing to do with ETW).     

So ideally we do not couple them here.   However I am OK with fixing this after checkin.